ddl/ddl_test.go

```diff
@@ -96,21 +98,24 @@ func TestT(t *testing.T) {
 	testleak.AfterTestT(t)()
 }
 
-func testNewDDLAndStart(ctx context.Context, c *C, options ...Option) *ddl {
+func testNewDDLAndStart(ctx context.Context, t *testing.T, options ...Option) *ddl {
 	// init infoCache and a stub infoSchema
 	ic := infoschema.NewCache(2)
 	ic.Insert(infoschema.MockInfoSchemaWithSchemaVer(nil, 0), 0)
 	options = append(options, WithInfoCache(ic))
 	d := newDDL(ctx, options...)
 	err := d.Start(nil)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
+
+	_, err = infosync.GlobalInfoSyncerInit(context.Background(), "t", func() uint64 { return 1 }, nil, true)
```

Comment:
I may be overlooking something, but I had to add this initializer somewhere, otherwise, the Table* tests would fail after migrating it to Testify.

My guess is that `infosync` on this context was already being initialized somewhere else, but I could not figure it out... the other place I see is the `TestT` function in this same file, but that is not being used by the Table* tests.